doc_requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx>=1.3.1
+sphinx_bootstrap_theme>=0.4.5
+sphinxcontrib-napoleon>=0.3.5
```

Comment:
I think you want to add `-r requirements.txt` here and give rtfd this as the requirements to install.  That is how we've done it in the past.   That should simplify the tox build a bit too I think